APIs/Subscriptions.qmd

```diff
@@ -0,0 +1,477 @@
+---
+title: Subscriptions 
+---
+# Push subscriptions
```

Comment:
If you write the title in the yml at the top, then it is suggested not to again include the title below in the markdown, i.e. # Push subscriptions. If needed, you can make it heading 2 with ##

Reply:
@Pratichhya I've updated it within the document

Reply:
@pjasiakcf Hi Paulina, I am trying to fix the Subscription Entity description table output using: https://quarto.org/docs/authoring/tables.html

Reply:
@Slegersj thank you for your help with the table. I see that there is still an issue with formatting within the table and that words are splitted in a column. Can we try to fix it somehow? We can also try to change the width of the columns.

Reply:
@pjasiakcf I will check as soon as possible if the table settings used [here](https://documentation.dataspace.copernicus.eu/Quotas.html#copernicus-general-users) can help us.

Reply:
Hi @Slegersj, now I see that we have the conflict in the branch and the preview still shows the wrong ver. of the table.

Reply:
Hi @pjasiakcf I resolved the conflict in the branch and you can check the new formatting of the table.

Reply:
Hi @Slegersj  still I see the words splitted in the first column. Is it possible to fix? If not I think we can publish it as it is. It looks much better now.